platform/ui/src/contextProviders/DialogProvider.styl

```diff
@@ -1,8 +1,8 @@
-.DraggableItem
+.DraggableItem.draggable
   div
     cursor: grab !important
 
-.DraggableItem.dragging
+.DraggableItem.draggable.dragging
```

Comment:
If it's a DraggableItem, why wouldn't it already be `.draggable`?

Reply:
This is because a draggable item can have its dragging functionality deactivated for a while if needed.